eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -126,23 +126,29 @@ jobs:
         Linux:
           OSVmImage: "ubuntu-18.04"
           TestTargetFramework: netcoreapp2.1
+          CodeCoverageArgs: ''
```

Comment:
We should default this at the higher level instead of passing empty to all entries.